package.json

```diff
@@ -21,16 +21,21 @@
   ],
   "files": [
     "wikimedia.json",
-    "qunit.json"
+    "qunit.json",
+    "services.json"
   ],
   "author": "Gaidarenko Oleg <markelog@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "eslint-plugin-qunit": "^3.3.0"
+    "eslint-plugin-qunit": "^3.3.0",
+    "eslint-plugin-jsdoc": "^3.2.0",
+    "eslint-plugin-json": "^1.2.0"
```

Comment:
Technically `eslint` itself is a peerDependency, so if we're specifying them - we need to specify all of them. However, it's really inconvenient - all the peerDependencies must be listed in all the users `package.json` and the  versions must match.  I'm wondering if it would be better to make these regular dependencies, so that clients can simply depend on the config.

Alternatively, we could list the peerDependencies here. and make a proxy package, like `wikimedia-eslint` that would work as an aggregator for peer dependencies.

I li ke the former solution for simplicity even though it's not entirely correct from a philosophical perspective.